src/Strategy/Node.hs

```diff
@@ -24,10 +24,11 @@ import Control.Effect.Diagnostics (
   recover,
   warnOnErr,
  )
+import Control.Effect.Reader (Reader)
 import Control.Monad (void, (<=<))
 import Data.Glob (Glob)
 import Data.Glob qualified as Glob
-import Data.List.Extra (singleton) -- singleton is in Data.List in base 4.16
```

Comment:
Had to remove this comment, it was causing failed idempotence checks in the formatter.  It's also not a major concern, since it will be a name conflict once we upgrade to GHC 9.2 and `base-4.16`.